lib/zlib.js

```diff
@@ -816,14 +815,8 @@ function Brotli(opts, mode) {
     new binding.BrotliDecoder(mode) : new binding.BrotliEncoder(mode);
 
   this._writeState = new Uint32Array(2);
-  // TODO(addaleax): Sometimes we generate better error codes in C++ land,
-  // e.g. ERR_BROTLI_PARAM_SET_FAILED -- it's hard to access them with
-  // the current bindings setup, though.
```

Comment:
Please don't remove this TODO comment until it's actually addressed :)

Reply:
I think I misunderstood the TODO in here. Can you explain if you don't mind?

Reply:
Yeah, the TODO is about surfacing the actual error codes we get from the underlying library, instead of a generic "initialization failed" error code – `ERR_BROTLI_PARAM_SET_FAILED` being one example of that

Reply:
I reverted the change. Thank you for the review.

Reply:
I moved it. There is 2 occurrences where we throw the error on C++. For the sake of reducability, I just kept the TODO on one of them.